movieapp/src/containers/FilterElement/FilterElement.tsx

```diff
@@ -0,0 +1,44 @@
+import { FilterElement } from './FilterElementTypes';
+import { Option } from '../../components/Option/Option';
+import { useDispatch } from 'react-redux';
+import { receiveMoviesFromApiError, sortMoviesById } from '../../actions/actions';
+import { AnyAction } from 'redux';
+
+const movieSortCriterias: Array<FilterElement> = [
+  {
+    id: '1',
+    title: 'Drama',
+    value: 'drama'
+  },
+  {
+    id: '2',
+    title: 'Comedy',
+    value: 'comedy'
+  },
+  {
+    id: '3',
+    title: 'Animation',
+    value: 'animation'
+  }
+];
+
+export function FilterElement(): JSX.Element {
+  const dispatch = useDispatch();
+
+  function FilterMovies(e: any): void {
+    fetch('http://localhost:4000/movies?searchBy=genres&filter=' + e.currentTarget.value + '&sortOrder=asc')
```

Comment:
creation of the query string should be as pure function from utils folder.

e.g. 
```
function searchParamsToQueryString (searchParams): String => {}

```
usage: 
```
const query = searchParamsToQueryString({
  searchBy: 'genres',
  sortOrder: 'asc'
}) // '?searchBy=genres&sortOrder=asc'
```

Reply:
Fixed